components/movementsensor/movementsensor.go

```diff
@@ -154,3 +155,18 @@ func DefaultAPIReadings(ctx context.Context, g MovementSensor, extra map[string]
 
 	return readings, nil
 }
+
+// UnimplementedAccuracies returns accuracy values that will not show up on movement sensor's RC card
+// or be useable for a caller of the GetAccuracies method. The RC card currently continuosly polls accuracies,
+// so a nil error must be rturned from the GetAccuracies call.
```

Comment:
Typo: returned

Not sure how the conclusion follows from the premise: if the RC card didn't continuously poll, would we not still need a nil error? I thought that was just the type signature that fit into the larger framework no matter what.

Reply:
The nil error stops the RC card from showing and continuously spams the server with logs. The true fix on the RC side is to handle errors from the GetAccuracy call gracefully so you can see your movement sensor data and hide the Accuracy tabs under each movement sensor data type. It'll be handled by Fleet. This is my fix to get the RC in good working order for all the other movement sensors that do not report accuracy.